test/models/system_test.rb

```diff
@@ -168,6 +168,10 @@ def test_with_installable_errata
 
       systems = System.with_installable_errata([non_installable.first])
       refute systems.include?(@errata_system)
+
+      systems = System.with_installable_errata([installable.first, non_installable.first])
+      assert_include systems, @errata_system
+      assert_equal systems.length, 1
```

Comment:
Based on what this intended to fix, I think an assertion showing that a system that has errata from `installable` does not get returned when that is combined with the `non_installable` errata (that would in theory, also be applicable to the system).